.travis.yml

```diff
@@ -6,9 +6,9 @@ sudo: false
 cache:
   directories: [$HOME/.cache/pip]
 
-install: pip install sphinx~=1.3.0 git+https://github.com/fabpot/sphinx-php.git
+install: pip install -r _build/.requirements.txt sphinx==1.3.0
```

Comment:
@javiereguiluz The requirement of `1.3.0` for TravisCI is due to the following warning (turned into error) that halts the build due to `sphinx-php` overwriting `code-block` using `sphinx` versions greater than `1.3`:

> Warning, treated as error:
WARNING: while setting up extension sensio.sphinx.codeblock: directive 'code-block' is already registered, it will be overridden

We may need to re-think the use of `-nW` opts for Travis runs, as this is incompatible with newer versions of Sphinx, and this will become a problem at some point (as we being to reply on newer features and **need** to run newer versions).